tables_cleaner/management/commands/clean_tables.py

```diff
@@ -34,6 +34,10 @@ def add_arguments(self, parser):
             '--database', action='store', dest='database', default=DEFAULT_DB_ALIAS,
             help='Nominates a specific database to load fixtures into. Defaults to the "default" database.',
         )
+        parser.add_argument('--model', action='store', dest='model', help="Process one app.model")
+        parser.add_argument('--keep-record-count', action='store', dest='record_count', help="Keep specified number of records (default: use settings)")
```

Comment:
`--keep-records` feels a bit ambiguous.

Also do you want to set a max line length ?

Feel free to rip the PR apart and i'll update it accordingly.